cmd/sync/sync.go

```diff
@@ -54,4 +74,20 @@ func main() {
 	if err := sync(); err != nil {
 		panic(err)
 	}
+	if *once {
+		return
+	}
+
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, os.Interrupt, syscall.SIGTERM)
+	for {
+		select {
+		case <-time.After(30 * time.Second):
```

Comment:
I think this is too short. Seen sometimes sync taking more than minute to complete.

Reply:
Agreed, I was planning on extending to 2-3 minutes.